test/setup/setup.spec.js

```diff
@@ -0,0 +1,5 @@
+describe('setup', () => {
```

Comment:
It's just checks that `test/setup.js` is included and mocking globals is working. So, if any migration to another unit testing library will be in the future, you will never miss to require `test/setup.js` :)